root/report/AsinsWithMultipleReleases.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+
+import ReleaseUrlList from './components/ReleaseUrlList';
+import FilterLink from './FilterLink';
+import type {ReportDataT, ReportReleaseUrlT} from './types';
+
+const AsinsWithMultipleReleases = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseUrlT>) => (
+  <Layout fullWidth title={l('Amazon URLs linked to multiple releases')}>
+    <h1>{l('Amazon URLs linked to multiple releases')}</h1>
+
+    <ul>
+      <li>
+        {exp.l(`This report shows Amazon URLs which are linked to multiple
+            releases. In most cases Amazon ASINs should map to MusicBrainz
```

Comment:
`releases` should be vertically aligned with `This`, and so on for the following lines, happens in some other files where `exp.l` is used too.